### ISSUE TITLE
Normalization issue with uint8 images in 

### ISSUE THREAD

**Issue by alexjtaylor** [aiff22/PyNET-PyTorch#10]
In the code below, it appears that the intention is to scale 10-bit raw data in the 16-bit PNG encodings of the original DNG images into the floating point range [0,1]. However, looking through the input patched images in train/huawei_raw/, there is a mix of uint8 images and uint16 images where it appears that in the uint8 case, the some of the images in the training set have been scaled into the range [0,255], instead of the [0,1024] expected for the original 10-bit data.

```
def extract_bayer_channels(raw):

    # Reshape the input bayer image

    ch_B  = raw[1::2, 1::2]
    ch_Gb = raw[0::2, 1::2]
    ch_R  = raw[0::2, 0::2]
    ch_Gr = raw[1::2, 0::2]

    RAW_combined = np.dstack((ch_B, ch_Gb, ch_R, ch_Gr))
    RAW_norm = RAW_combined.astype(np.float32) / (4 * 255)

    return RAW_norm
```
Should dtype np.uint8 inputs in raw instead be scaled by 255 instead of 4*255 so that inputs to the network will be consistently scaled?

**Comment by alexjtaylor** [comment]
Nevermind, I believe that this is in fact correct and that some of the patches were in the original range [0,255] in 10-bit, so no rescaling was done, just uint8 used where possible to reduce size.
